app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.errors import VaccineError, NotWearingMaskError
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
```

Comment:
It's a good practice to use type hinting in Python. However, it would be better if you could specify the type of the elements in the list `friends`. For example, if `friends` is a list of `Person` objects, you can specify it like this: `friends: List[Person]`.